dashboard/app/views/levels/_admin.html.haml

```diff
@@ -46,6 +46,7 @@
                 = text_field_tag :name, @level.name
                 = hidden_field_tag :authenticity_token, form_authenticity_token
                 = submit_tag 'Clone'
+              .validation-error{style: 'color: red;'}
```

Comment:
this works because of the logic here:  https://github.com/code-dot-org/code-dot-org/blob/1cad5ac667f9ed1a3aa31063e6d06bd21a78e2bd/apps/src/code-studio/ajaxSubmit.js#L27-L29

which is the logic that was written to show this error on the /levels page: https://github.com/code-dot-org/code-dot-org/blob/ee739df29c908f316e621af3e19bd5886c1d414d/dashboard/app/views/levels/_list.html.haml#L104
![Screen Shot 2022-04-27 at 1 13 48 PM](https://user-images.githubusercontent.com/8001765/165623303-e579c1f2-d9f7-44ea-abcb-168f9519875e.png)